classes/gateways/class.pmprogateway_stripe.php

```diff
@@ -3396,4 +3422,20 @@ private function convert_price_to_unit_amount( $price ) {
 
 		return intval( $price * $currency_unit_multiplier );
 	}
+	/**
+	 * Determine whether the calling method was called from
+	 * inside or outside of the Stripe Gateway class.
+	 *
+	 * Useful for preparing to change method visibility from
+	 * public to private.
+	 *
+	 * @return bool
+	 */
+	static private function method_being_called_from_outside_class() {
+		$backtrace = debug_backtrace();
```

Comment:
```suggestion
		$backtrace = debug_backtrace( DEBUG_BACKTRACE_IGNORE_ARGS );
```

Reply:
If we don't need arguments we don't need to get them, this just saves a little bit of memory